src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useRef } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function Form({ savedList, setSavedList }) {
+  const [input, setInput] = useState([]);
+  const onChange = (e) => {
```

Comment:
이벤트 핸들러의 이름은 보통 `handle + 대상 + 동작`의 형식으로 짓습니다.

```suggestion
  const handleInputChange = (e) => {
```

Reply:
참고하여 수정하겠습니다!!